src/modules/scripts.nix

```diff
@@ -1,22 +1,49 @@
-{ pkgs, lib, config, ... }:
+{
+  lib,
+  config,
+  pkgs,
+  ...
+}:
 
 let
   types = lib.types;
-  scriptType = types.submodule ({ config, ... }: {
-    options = {
-      exec = lib.mkOption {
-        type = types.str;
-        description = "Bash code to execute when the script is run.";
-      };
-      description = lib.mkOption {
-        type = types.str;
-        description = "Description of the script.";
-        default = "";
+  scriptType = types.submodule (
+    { config, ... }:
+    {
+      options = {
+        exec = lib.mkOption {
+          type = types.str;
+          description = "Shell code to execute when the script is run.";
+        };
+        interpreter = lib.mkOption {
```

Comment:
I feel like we should call it `package` since we're calling it that way everywhere else too.

Reply:
No problem. Are you happy with binary, or is there a convention there too? Bin?

Reply:
Fixed